CONTRIBUTING.md

```diff
@@ -14,17 +14,17 @@ To get an overview of the project, read the [README](README.md). Here are some r
 
 #### Create a new issue
 
-If you spot a problem with minitz, [search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments). If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/hexagon/minitz/issues/new/choose). 
+If you spot a problem with minitz, [search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments). If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/hexagon/minitz/issues/new/choose).
 
 #### Solve an issue
 
-Scan through our [existing issues](https://github.com/hexagon/minitz/issues) to find one that interests you. You can narrow down the search using `labels` as filters. If you find an issue to work on, make a note in the comments so er van assign it to you. Then you are welcome to open a PR with a fix.
+Scan through our [existing issues](https://github.com/hexagon/minitz/issues) to find one that interests you. You can narrow down the search using `labels` as filters. If you find an issue to work on, drop a note in the comments asking to assign it to you. After that, you can start with your changes and open a PR with a fix.
 
 ### Make Changes
 
 #### Setting up the environment
 
-Se recommend using VS Code with eslint extensions, which will automatically check your code against THE defined rules as you write it.
+We recommend using VS Code with eslint extensions, which will automatically check your code against the defined rules as you write it.
```

Comment:
Recommendation on a little rewording: "We recommend using VS Code with eslint extensions. It will automatically check your code against the defined rules."